spec/support/request/stripe_stubs.rb

```diff
@@ -170,7 +171,8 @@ def get_customer_payment_methods_response_mock(options)
     fingerprint = options[:fingerprint] || "7890"
     { status: options[:code] || 200,
       body: JSON.generate(
```

Comment:
(breaking suggestion into two parts for GitHub)
The indentation here doesn't look right. How about we break it up further:

```suggestion
    { 
      status: options[:code] || 200,
      body: JSON.generate(
```